db/migrate/20170403093355_add_non_visible_id_label_to_online_verifications_label.rb

```diff
@@ -0,0 +1,10 @@
+class AddNonVisibleIdLabelToOnlineVerificationsLabel < ActiveRecord::Migration[5.0]
+  def up
+    OnlineVerifications::Label.create(code: 'non_visible_id')
+  end
+
+  def down
+    ov_label = OnlineVerifications::Label.find_by_code('non_visible_id')
+    ov_label.delete
+  end
```

Comment:
I normally prefer to use plain SQL in migrations. That way the migration is resilient to model renaming (see the [original migration](https://github.com/EnComu/participa/blob/master/db/migrate/20170329135829_create_online_verification_labels.rb) for an example). Another popular solution for this is to create an empty class before the migration with the name you'll be using.

```
class Onlineverifications::Label < Applicationrecord; end
```

That way you ensure that the class is defined (even if it was renamed at migration time) and you can use all the Activerecord niceties and don't need to use SQL-only constructs like `now()`.

This PR will work ok, so feel free to merge if you don't have time for this and we can improve after merge.